doc/index.txt

```diff
@@ -21,6 +21,9 @@ learning/machine learning <https://mila.umontreal.ca/en/cours/>`_ classes).
 News
 ====
 
+* Removed support for the old (device=gpu) backend.  Use the new
+  backend (device=cuda) for gpu computing.
+
```

Comment:
Tell that it is removed from the dev version and link to https://github.com/Theano/Theano/wiki/Converting-to-the-new-gpu-back-end%28gpuarray%29